microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview of Manifest V3
-description: Migrating from Manifest V2 to V3 will help reduce fragmentation of the web for developers and enhance privacy, security, and performance for end users.
+title: Overview and timelines for migrating to Manifest V3
```

Comment:
Please change ALL places.
* "title: " yaml fileld
* h1 `#` title a few lines below it.
* toc.yml title
* full-text search this repo for the old name, update all link text.

Plan on a couple passes of this while you work with an Editor to fine-tune the new title.  It makes sense for either the PM/Dev or the Writer/Editor to make this mechanical change - there's a pretty strong argument for having Docs team handle this mechanics implementation.  @vchapel

Reply:
resolved?

Reply:
Thanks for reviewing and updating Victor and Mike. Is this resolved?

Reply:
Resolved; "title" and h2 & TOC match.